examples/standalone-controls.html

```diff
@@ -26,7 +26,7 @@
       <h1>Standalone Controls Example</h1>
       <p>
         This example shows how individual controls can be used without the
-        <code>media-container</code> element.
+        <code>media-controller</code> element.
```

Comment:
This change is unrelated. Just noticed that this still referenced `media-container`.